src/ui/js/Overview.js

```diff
@@ -213,25 +211,37 @@ Overview.pageAddGameTable = function(
     sectionHeader,
     reverseSortOrder
   ) {
-  if (gameType == 'new') {
-    Overview.pageAddGameTableNew();
-    return;
-  }
-
-  if (typeof reverseSortOrder === 'undefined') {
-    reverseSortOrder = false;
-  }
-
   var gamesource;
   var tableClass;
-  if (gameType == 'finished') {
-    gamesource = Api.completed_games.games;
-    tableClass = 'finishedGames';
-  } else if (gameType == 'rejected') {
-    gamesource = Api.rejected_games.games;
-    tableClass = 'rejectedGames';
-  } else {
-    gamesource = Api.active_games.games[gameType];
+
+  switch(gameType) {
```

Comment:
FTR, it looks like we're not consistent about horizontal whitespace after switch --- i'd prefer we standardize on having some, because i like horizontal whitespace for everything that's not a function (so more like `switch (gameType) {`.

Obviously, i won't block the pull over this, just stating a preference for the future.